### ISSUE TITLE
follow your instructions but wrong answer

### ISSUE THREAD

**Issue by lyxlynn** [Engineering-Course/LIP_SSL#20]
Hi , I followed your config and data, training from initial state and using LIP_train_images,but the loss always be around 5~10.  I use the final caffemodel to  test but all the outputs are zeros.

**Comment by Engineering-Course** [comment]
You should use a pre-trained model.
